src/core/logging/logger.ts

```diff
@@ -10,6 +11,7 @@ export class Logger {
 	private botConfig: BotConfig;
 	public discordLogger?: DiscordLogger;
 	public slackLogger?: SlackLogger;
+	private telegramLogger?: TelegramLogger; 
 
 	// Codes that are not sent to external sources (discord, slack)
 	private externalExemptCodes: Array<number> = [4, 5, 6, 8];
```

Comment:
can we add this to env var as well please, maybe in the new external logging config or in the skip config.